models/tasks.js

```diff
@@ -193,6 +194,10 @@ const fetchPaginatedTasks = async ({
         .where("title", "<=", title + "\uf8ff")
         .orderBy("title", "asc");
     }
+    if (assigneeRole) {
+      const archivedUserIds = await userUtils.getArchivedUserIds();
+      initialQuery = initialQuery.where("assignee", "in", archivedUserIds);
+    }
```

Comment:
Where are we checking what is passed in `assigneeRole` query?

Reply:
added checks now